<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QMainWindow" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HomePage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(243, 248, 254)</string>
   </property>
   <widget class="QGraphicsView" name="top">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>25</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string>欢 迎 使 用 航 空 购 票 系 统 ！</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="bottom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>800</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);
background-color: rgb(85, 170, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homeBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
    background-color: rgba(255, 255, 255, 150); 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="flight_images.qrc">
      <normaloff>:/sign/homepage.png</normaloff>:/sign/homepage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>80</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buyButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="flight_images.qrc">
      <normaloff>:/sign/buyTicketPage.png</normaloff>:/sign/buyTicketPage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>80</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="staffCenterButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="flight_images.qrc">
      <normaloff>:/sign/customerServicePage.png</normaloff>:/sign/customerServicePage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>80</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="userInfoButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="flight_images.qrc">
      <normaloff>:/sign/personinfoPage.png</normaloff>:/sign/personinfoPage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>80</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="picture1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>361</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="flight_images.qrc">:/sign/xi'an.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="picture2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>160</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="flight_images.qrc">:/sign/xi'zang.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="picture3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>160</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="flight_images.qrc">:/sign/xin'jiang.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>西安：古丝绸之路的起点，历史的见证</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>西藏：世界屋脊，心灵的净土</string>
    </property>
   </widget>
   <widget class="QLabel" name="text3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>新疆：丝绸之路上的神秘土地</string>
    </property>
   </widget>
   <widget class="QWidget" name="city" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(180, 211, 250,200);
border-radius: 10px;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>热门城市</string>
     </property>
    </widget>
    <widget class="QLabel" name="hot1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>🔥西安</string>
     </property>
    </widget>
    <widget class="QLabel" name="hot2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>🔥西藏</string>
     </property>
    </widget>
    <widget class="QLabel" name="hot3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>🔥新疆</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ticket" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(180, 211, 250,200);
border-radius: 10px;</string>
    </property>
    <widget class="QLabel" name="ticketlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>特价机票</string>
     </property>
    </widget>
    <widget class="QWidget" name="discountticket" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,200);
border-radius: 10px;</string>
     </property>
     <widget class="QLabel" name="beginTime_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="fromPlace_2">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>40</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>A城市</string>
      </property>
     </widget>
     <widget class="QLabel" name="arrivalTime_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>12:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="airlineNumber_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>AB1234</string>
      </property>
     </widget>
     <widget class="QLabel" name="toPlace_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>B城市</string>
      </property>
     </widget>
     <widget class="QLabel" name="arrow">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>30</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
      </property>
      <property name="text">
       <string>&gt;&gt;&gt;&gt;&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ticket1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(85, 170, 255,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lowest">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);</string>
      </property>
      <property name="text">
       <string>最低               元</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);
</string>
      </property>
      <property name="text">
       <string>66</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
     </widget>
     <zorder>graphicsView</zorder>
     <zorder>beginTime_2</zorder>
     <zorder>fromPlace_2</zorder>
     <zorder>arrivalTime_2</zorder>
     <zorder>airlineNumber_2</zorder>
     <zorder>toPlace_2</zorder>
     <zorder>arrow</zorder>
     <zorder>lowest</zorder>
     <zorder>label_2</zorder>
     <zorder>ticket1</zorder>
    </widget>
    <widget class="QWidget" name="discountticket_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,200);
border-radius: 10px;</string>
     </property>
     <widget class="QLabel" name="beginTime_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="fromPlace_4">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>40</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>A城市</string>
      </property>
     </widget>
     <widget class="QLabel" name="arrivalTime_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>12:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="airlineNumber_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>AB1234</string>
      </property>
     </widget>
     <widget class="QLabel" name="toPlace_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>B城市</string>
      </property>
     </widget>
     <widget class="QLabel" name="arrow_3">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>30</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);</string>
      </property>
      <property name="text">
       <string>&gt;&gt;&gt;&gt;&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lowest_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);</string>
      </property>
      <property name="text">
       <string>最低               元</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);
</string>
      </property>
      <property name="text">
       <string>66</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ticket2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(85, 170, 255,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
     </widget>
     <zorder>graphicsView_2</zorder>
     <zorder>lowest_3</zorder>
     <zorder>beginTime_4</zorder>
     <zorder>fromPlace_4</zorder>
     <zorder>arrivalTime_4</zorder>
     <zorder>airlineNumber_4</zorder>
     <zorder>toPlace_4</zorder>
     <zorder>arrow_3</zorder>
     <zorder>label_4</zorder>
     <zorder>ticket2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="flight_images.qrc"/>
 </resources>
 <connections/>
</ui>
